test/c/pragma-once.h

```diff
@@ -0,0 +1,2 @@
+/** Ensure this file is parsed as a header */
+#pragma once
```

Comment:
Honestly, the whole error this would generate otherwise is a bit bogus if all it takes is to have `-xc*` or `-xc*-header`. Not sure we can rely on these tests failing long term is what I'm saying, so I'm not sure the tests are useful at all.

Not to mention it's a non standard pragma...

I understand why you're adding them and I won't oppose, but if it were only up to me, I wouldn't include these.

Reply:
Fair points. I was hoping to add some tests for the fixes, but this is probably a bit fragile indeed. Threw them away.